src/pydvl/utils/numeric.py

```diff
@@ -101,16 +96,31 @@ def random_powerset(
     if q < 0 or q > 1:
         raise ValueError("Element sampling probability must be in [0,1]")
 
+    rng = np.random.default_rng()
```

Comment:
We should add an optional `seed` value here to make this repeatable again. otherwise the previous version was better.

Reply:
Leaving as-is in view of #242